modules/application-health-about.adoc

```diff
@@ -102,50 +94,6 @@ spec:
 <4> A container command test.
 <5> The commands to execute on the container.
 
-.Sample container command startup probe and liveness probe with container command tests in a pod spec
-[source,yaml]
-----
-apiVersion: v1
-kind: Pod
-metadata:
-  labels:
-    test: health-check
-  name: my-application
-...
-spec:
-  containers:
-  - name: goproxy-app <1>
-    args:
-    image: k8s.gcr.io/goproxy:0.1 <2>
-    livenessProbe: <3>
```

Comment:
not worth to keep it for liveness probe? or is it only used for the startup probe variant in the text above?

Reply:
@sttts One of the other examples does include a liveness probe so I thought that would be sufficient. But if you'd prefer, we could modify the example to remove just the startup probe config. Let me know what you think.

Keep in mind too that this only applies to 4.4, and that all examples will be shown in 4.5+.

Reply:
am fine with removing it